### PR TITLE
Remoter: added ability to compeltely silence the remoter

### PR DESCRIPTION
**Pull request by bentsi**
When running remote command with emoter.run and setting  verbose=True and ignore_status=True that will now mean
that nothing will be printed in any case to log. This fixes too much logging issue with coredumpctl command
that returns error when no coredumps found, so we have got a lot of 'error' messages in the log
Task: https://trello.com/c/3bTFObwY/1197-coredumps-check-are-producing-too-much-logging-1194
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@bentsi this is to lower the prints from the coredump scanning ?

File: sdcm/remote.py [hunk 1]
```diff
@@ -248,13 +248,11 @@ def _run():
                 raise
             except Exception as details:  # pylint: disable=broad-except
                 if hasattr(details, "result"):
-                    self._print_command_results(details.result, verbose)   # pylint: disable=no-member
+                    self._print_command_results(details.result, verbose, ignore_status)   # pylint: disable=no-member
                 raise
 
         result = _run()
-        self._print_command_results(result, verbose)
-        #result.stdout = result.stdout.encode(encoding='utf-8')
```
Comment (fruch) [inline sdcm/remote.py]: my bad, that those comments are here...

**Comment by fruch** [comment]
what about filtering the db nodes for the relevant things ? the coredump generate a handful of prints each time, on each node.

**Comment by bentsi** [comment]
> @bentsi this is to lower the prints from the coredump scanning ?

Yes

**Comment by bentsi** [comment]
regarding the nodes, I don't think we can control what is printed in journalctl

**Comment by fruch** [comment]
we can filter what we are sending to our test, a bit complex but not much

**Pull request merged**